.github/dependabot.yml

```
@@ -1,8 +1,9 @@
+---
 version: 2
 updates:
-- package-ecosystem: "github-actions"
-  directory: "/"
-  schedule:
-    interval: daily
-    time: "06:00"
-  open-pull-requests-limit: 10
\ No newline at end of file
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

```suggestion
      interval: weekly
```
Let's do weekly for these